SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -160,8 +160,13 @@ private void populateTaskEmailProperties(Map<String, Object> templateProperties,
     Optional<String> directory = taskManager.getDirectory(taskId);
 
     templateProperties.put("singularityTaskLink", getSingularityTaskLink(taskId));
-    templateProperties.put("stdout", getTaskLogFile(taskId, "stdout", task, directory).or(new String[0]));
-    templateProperties.put("stderr", getTaskLogFile(taskId, "stderr", task, directory).or(new String[0]));
+
+    Map<String, String[]> logTails = Maps.newHashMap();
```

Comment:
`Map`s are unordered -- use `new LinkedHashMap<>()` in order to enforce the log file ordering here